i18n/en.json

```diff
@@ -2816,6 +2816,9 @@
   "post_info.reply": "Reply",
   "post_info.system": "System",
   "post_info.unpin": "Un-pin from channel",
+  "post_info.tooltip.add_reactions": "Add Reactions",
```

Comment:
Not important because it will be automatically solved soon, but It should be in alphabetical order inside the file.

Reply:
Yeah I just let it do that automatically.